src/moments.jl

```diff
@@ -1,52 +1,70 @@
 ##### Weighted var & std
 
 ## var
-
 """
-    varm(x, wv::WeightVec, m, [dim])
-
-Return the variance of a real-valued array `x` with a known mean `m`, optionally
-over a dimension `dim`. The weighting vector `wv` specifies frequency weights
-(also called case weights) for the result.
-
-This function differs from its counterpart in Base in that Bessel's correction
-is not used. That is, here the denominator for the variance is `sum(wv)`,
-whereas it's `length(x)-1` in `Base.varm`. The impact is that this is not a
-weighted estimate of the population variance based on the sample; it's the weighted
-variance of the sample.
+    varm(x, w::AbstractWeights, m, [dim]; corrected=false)
+
+Compute the variance of a real-valued array `x` with a known mean `m`, optionally
+over a dimension `dim`. Observations in `x` are weighted using weight vector `w`.
+The uncorrected (when `corrected=false`) sample variance is defined as:
+```math
+\\frac{1}{\\sum{w}} \\sum_{i=1}^n {w_i\\left({x_i - m}\\right)^2 }
+```
+where ``n`` is the length of the input. The unbiased estimate (when `corrected=true`) of
+the population variance is computed by replacing
+``\\frac{1}{\\sum{w}}`` with a factor dependent on the type of weights used:
+* `AnalyticWeights`: ``\\frac{1}{\\sum w - \\sum {w^2} / \\sum w}``
+* `FrequencyWeights`: ``\\frac{1}{\\sum{w} - 1}``
+* `ProbabilityWeights`: ``\\frac{n}{(n - 1) \\sum w}`` where ``n`` equals `count(!iszero, w)`
```

Comment:
Ah, and it would be cool to mention that no correction is possible for generic `Weights`.